styles/theme.ts

```diff
@@ -138,6 +148,37 @@ let theme = createTheme({
       styleOverrides: {
         ...buttonStyleOverrides,
       },
+      variants: [
```

Comment:
Is this common for every case in our application ?

Reply:
yes